api/v1alpha3/etcdadmconfig_types.go

```diff
@@ -50,12 +58,16 @@ type EtcdadmConfigSpec struct {
 	// +optional
 	Version string `json:"version,omitempty"`
 
-	// EtcdReleaseURL is an optional field to specify where etcdadm can download etcd from
+	// ImageRepository is an optional field to specify where etcdadm can pull etcd images from
 	// +optional
-	EtcdReleaseURL string `json:"etcdReleaseURL,omitempty"`
```

Comment:
etcdadm uses this as one of the args when running downloading etcd for running it as a systemd service so we shouldn't remove this

Reply:
it's back :)